examples/nucleo_f091rc/blink/main.cpp

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2021, Thomas Sommer
```

Comment:
Sure?

Reply:
It's an unmodified copy of the NUCLEO-F072RB blink/main.cpp. I mean the copyright of our copy-paste blinky examples is a little dumb anyways…

Reply:
(you're still right tho)